apps/studio/components/layouts/ProjectLayout/NavigationBar/NavigationBar.tsx

```diff
@@ -131,6 +131,7 @@ const NavigationBar = () => {
             }}
             onClick={onCloseNavigationIconLink}
           />
+          
```

Comment:
can you remove empty lines from your PR? This helps in reviewer to focus on the correct changes